docs/src/main/paradox/recordio.md

```diff
@@ -0,0 +1,61 @@
+# RecordIO Framing
+
+The codec parses an incoming RecordIO-formatted stream of ByteStrings into distinct frames.
```

Comment:
@catalin-ursachi Can you add the remaining clarification and/or link, so that we can merge this?